### ISSUE TITLE
show stack of volumes with scipion

### ISSUE THREAD

**Issue by mmmtnez** [I2PC/scipion#1505]
1) execute test
    scipion test tests.em.protocols.test_protocols_import_volumes.TestImportVolumes
2) the test import volumes, set of volumes, etc
3) if we open the set of volumes imported by the protocol labelled "import 2 volumes" and we press the analyze button and then the slide option both volumes are available (we may switch between them using the volume drop menu"
4) do the same with the import protocol labelled import 3 volumes from stack. Press analyze and slices. The drop menu "volume" have disappeared. If we maximize the window the menu reappears and we may select any of the three volumes. In this second case the name of the volumes in the menu shows the absolute path, in the previous case just the volume name.

In summary scipion does not allow to change between the volumes contained by a stack if the absolute path to the file does not fit in the show window

**Comment by pconesa** [comment]
@mmmtnez, I'm not getting this behaviour, it might be combined with some low screen resolution?
![image](https://user-images.githubusercontent.com/785633/63494718-968a7f00-c4be-11e9-8032-e173a9177966.png)

**Comment by pconesa** [comment]
I managed to get it increasing the path length.
